src/main/java/uk/gov/hmcts/reform/managecase/repository/DataStoreRepository.java

```diff
@@ -22,9 +22,10 @@
 
     void removeCaseUserRoles(List<CaseUserRole> caseUserRoles, String organisationId);
 
-    CaseViewResource findCaseByCaseId(String caseId);
+    CaseViewResource findCaseByCaseId(String caseId, String userAuthToken);
 
-    CaseResource submitEventForCase(String caseId, String eventId, ChangeOrganisationRequest changeOrganisationRequest);
+    CaseResource submitEventForCase(String caseId, String eventId, ChangeOrganisationRequest changeOrganisationRequest,
+                                    String userAuthToken);
 
     CaseResource findCaseByCaseIdExternalApi(String caseId);
 }
```

Comment:
> CaseResource findCaseByCaseIdExternalApi(String caseId);
though this api doesn't need for approver.. probably worth to inline with other api signatures??

Alternatively define two specific api's wherever required and abstract userToken population inside repo.

CaseViewResource findCaseByCaseIdForCaa(String caseId);
CaseViewResource findCaseByCaseIdForApprover(String caseId);